### ISSUE TITLE
Add tests for METS file parsing process

### ISSUE THREAD

**Issue by jraddaoui** [CCA-Public/scope#73]
Currently, there are no tests for https://github.com/CCA-Public/dip-access-interface/blob/master/accesspoc/dips/parsemets.py

This class is being re-formatted in #70, which should ease the addition of such tests.

**Comment by jraddaoui** [comment]
Estimate: 8 hours.

**Comment by jraddaoui** [comment]
Current coverage report:

```
Name                                                     Stmts   Miss  Cover
----------------------------------------------------------------------------
dips/__init__.py                                             0      0   100%
dips/api_urls.py                                             3      0   100%
dips/api_views.py                                           24      0   100%
dips/apps.py                                                 5      0   100%
dips/forms.py                                               78     11    86%
dips/handlers.py                                             7      0   100%
dips/helpers.py                                             46      0   100%
dips/migrations/0001_initial.py                             11      0   100%
dips/migrations/0002_initial_data.py                        36      3    92%
dips/migrations/0003_alter_digitalfile_datemodified.py       4      0   100%
dips/migrations/0004_alter_digitalfile_puid.py               4      0   100%
dips/migrations/0005_alter_dublincore_language.py            4      0   100%
dips/migrations/0006_alter_dip.py                            5      0   100%
dips/migrations/0007_delete_taskresult.py                    4      0   100%
dips/migrations/0008_alter_dip_objectszip.py                 4      0   100%
dips/migrations/__init__.py                                  0      0   100%
dips/models.py                                             215      3    99%
dips/parsemets.py                                          157    138    12%    ***
dips/tasks.py                                               65      0   100%
dips/templatetags/__init__.py                                0      0   100%
dips/templatetags/custom_tags.py                            18      0   100%
dips/views.py                                              307     16    95%
manage.py                                                    6      0   100%
scope/__init__.py                                            2      0   100%
scope/celery.py                                              8      1    88%
scope/helpers.py                                            14      0   100%
scope/migrations/__init__.py                                 0      0   100%
scope/settings.py                                           51      0   100%
scope/staticfiles.py                                         3      0   100%
scope/urls.py                                                4      0   100%
search/__init__.py                                           0      0   100%
search/apps.py                                               7      0   100%
search/documents.py                                         30      0   100%
search/helpers.py                                           23      0   100%
search/management/__init__.py                                0      0   100%
search/management/commands/__init__.py                       0      0   100%
search/management/commands/index_data.py                    29      3    90%
search/tasks.py                                             45      0   100%
----------------------------------------------------------------------------
TOTAL                                                     1219    175    86%
```

Addressing this issue, this project could have around 97% code coverage.

**Comment by jraddaoui** [comment]
Addressed in #191, including a minor change to support the new event detail location in PREMIS v3 events from AM 1.10 and 1.11 METS files.
